src/bundles/SellerRegistration/components/PricingForm/PricingForm.js

```diff
@@ -73,6 +73,12 @@ class PricingForm extends BaseForm {
             {Object.keys(validServices).map((service, i) => (
               <fieldset key={`pricing.${service}.${i}`} className="field">
                 <legend>{service}</legend>
+                <label>
+                  Note: 95% of sellers submit bids below 
+                  ${Number(domains[service] ? domains[service] : 0).toFixed(2)}. 
```

Comment:
`(parseInt(domains[service])  || 0).toFixed(2)` is an alternative

Reply:
I used parseFloat based on the data type in the DB.